migrate/20251206_add_location_azs.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:location) do
+      add_column :azs, :jsonb, null: false, default: "[]"
```

Comment:
I agree with @serprex. Unless you have a good reason to want `jsonb`, a text array is  simpler.  However, the `default` needs to be different:

```suggestion
      add_column :azs, :jsonb, null: false, default: "{}"
```